learning-team/2020-07/2020-07-07.md

```diff
@@ -0,0 +1,108 @@
+- [x] Jen Weber (JW)
+- [ ] Jessica Jordan (JJ)
+- [ ] Kenneth Larsen (KL)
+- [ ] Melanie Sumner (MS)
+- [x] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+
+### Guests
+- [x] James Davis
+
+## Weekly review
+
+### Review action items (weekly)
+See: [Action items board](https://github.com/orgs/ember-learn/projects/47)
+
+- locks has merged a script for doing guides releases!
+- locks has a first version of a bot for the releases page
+- CM and JG have continued working on the discord bot to remind people of releases. Will share updates!
+
+### Review Initiatives board (every other week)
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/19)
+
+- CM - in the process of integrating the Discourse Comments into the blog. But it doesn't fit due to theming of the forum. So we added a new card to the website redesign to update the Discourse forum theme.
+
+### Review PR queue
+
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+### release scripts and automation (RM, 15mins)
+
+RM: There are 3 things to address here.
+- automating releases for each repository. We have done some of that.
+- making sure that we know how to release the thing if the automation fails or we have a special case
+- a tool to do the entire release
+
+Want to bring this up to the group because of something CM brough up. The tool is written in Rust because Node scripts are a pain in comparison. On the Guides source repo, the automation is in bash since it's the most portable. Some people may not be familiar but it's the lingua franca and requires no extra installs. API docs are Node scripts. I'm bringing this up so that we can discuss what should be written and how.
+
+CM: For truly internal tools, we have more flexibility. For things we want to be able to allow people to contribute to, we should use JavaScript. There are tools like shell.js that can help here. The current situation for the guides being difficult to release are due to unfinished work. There are project cards for it.
+
+RM: They are not hard to release, just they have some manual steps.
+
+CM: ideally it should be Ember deploy Algolia
+
+RM: We have to clear the Algolia index for each commit in master, right?
+
+CM/JW: no
+
+RM: Otherwise they would be out of date?
+
+CM: yes, and no. We generally do this only on releases. Otherwise we would run out of indexes. Do you mean the latest release or all of them?
+
+RM: The latest, because it keeps changing
+
+JW: I would not say no to donated work done in Rust. OSS is an opportunity for contributors to learn and grow, and that includes us. I would like if there was 1 other person who is active, who we can ask questions of. And also we could schedule a code walkthrough so that we can see how easy it would be for someone who doesn't know Rust to understand what is happening. I understand what you mean that Node can be difficult for scripting.
```

Comment:
```suggestion
JW: I would not say no to donated work done in Rust. OSS is an opportunity for contributors to learn and grow, and that includes us. I would like if there was 1 other person who is active, to whom we can ask [Rust] questions. And also we could schedule a code walkthrough so that we can see how easy it would be for someone who doesn't know Rust to understand what is happening. I understand what you mean that Node can be difficult for scripting.
```